pallets/xcm-core-buyer/src/lib.rs

```diff
@@ -349,56 +572,89 @@ pub mod pallet {
             // We use `descend_origin` instead of wrapping the transact call in `utility.as_derivative`
             // because with `descend_origin` the parathread tank account will pay for fees, while
             // `utility.as_derivative` will make the tanssi sovereign account pay for fees.
+
+            let notify_call = <T as Config>::RuntimeCall::from(Call::<T>::query_response {
+                query_id: 0,
```

Comment:
I guess this gets automatically set to a value other than 0 later?

Reply:
So, this `RuntimeCall` is sent across the destination chain and parameter's value does not matter as it will automatically be set to proper query id by destination chain.